### ISSUE TITLE
Custom search result rules have inconsistent results

### ISSUE THREAD

**Issue by jazzsequence** [humanmade/altis-enhanced-search#183]
When using custom search results, there are a number of unexpected inconsistencies in the returned behavior:

- Posts manually added to the search results don't appear in the actual search results
- Posts manually added to the search results are displayed as "reordered default" rather than "custom result"

Steps to reproduce:
1. Add custom synonyms `dungeons, dragons, dnd, d&d, dungeons & dragons`
2. Add posts containing "dungeons & dragons", "dnd" and "d&d"
3. Create a custom search result rule for "dungeons"
4. Add any other arbitrary post to the list -- the added post had the "RD" icon instead of the "CR" icon.
5. Perform a search -- the added post does not display

![Screen Shot 2021-04-07 at 2 20 06 PM](https://user-images.githubusercontent.com/991511/113929568-756ac500-97ad-11eb-9684-9eaf80a97083.png)
![Screen Shot 2021-04-07 at 2 19 49 PM](https://user-images.githubusercontent.com/991511/113929573-76035b80-97ad-11eb-971c-41462c50a2d9.png)

Expected result: 
Manually added post appears in the search results.

Acceptance criteria:

- [ ] ...

**Comment by roborourke** [comment]
Does the same thing happen on the front end?

**Comment by jazzsequence** [comment]
No, on the front end, the results are what's expected.

![Search Results for “dungeons” – Altis Demo Site](https://user-images.githubusercontent.com/991511/114098345-76b7f280-987e-11eb-904e-8dec9ac6b5fe.jpg)

**Comment by roborourke** [comment]
Thought that might be the case, presumably this is intentional but what's weird is I couldn't find the code that conditoinally only adds this to the front end. Unless EP just isn't being used in the admin search at the moment...

**Comment by jazzsequence** [comment]
> Unless EP just isn't being used in the admin search at the moment...

I don't think that's the case, because synonyms still work in the admin, as do other ES features (fuzzy searching tags and users, for example).

**Comment by roborourke** [comment]
Yeah we do want it to work in the admin. Ok so there's something in EP meaning custom search results don't work in the admin. I'd say this is relatively low priority.

**Comment by jazzsequence** [comment]
Maybe we should document it as a known issue and reprioritize/come back to it later?
